data/org.containers.hirte.internal.Agent.xml

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+<node>
+  <interface name="org.containers.hirte.internal.Agent">
+    <method name="StartUnit">
+      <arg name="name" type="s" direction="in" />
+      <arg name="job" type="o" direction="out" />
```

Comment:
Actually the internal functions I made ended up a bit different, as there is no real need to have a job object in the agent.